Survival Pokemon Center Heal Tracking hotfix

Fellshadow opened the pull request:
Added a little button that disables/enables the auto-tracking for PC Heals in Survival mode. It will only automatically increment while the button is showing "+" (Ideally a checkmark for full clarity but couldn't figure that out lol)

It's only toggleable via click currently (so no controller support yet)

Not the ideal end-scenario and not incredibly pretty but a quick temporary fix for #79 that would at least let the player delay the tracking until when the PC heals rule applies.

I'm also happy to work on adding buttons to adjust the count manually (my current thought is to have up/down chevrons/arrows (or similar) to the left of the number for this) and to improve this functionality, but submitting this now so that there's at least something if i don't get the rest done soon

Example Image: 
![image](https://user-images.githubusercontent.com/106463662/173559031-c09a22e0-68a7-46d6-98c4-63375454ba77.png)

(In addition i also restored the empty ROMS_FOLDER setting in Settings.ini as i noticed some users in the discord where confused about where to add it when it was just blank, i'm not sure if this change was to fix something though so lemme know if it's an issue)

Fellshadow commented:
@bdjeffyp tagging you for review

bdjeffyp commented on ironmon_tracker/Program.lua:
```diff
@@ -7,6 +7,7 @@ Program = {
 	trainerPokemonTeam = {},
 	enemyPokemonTeam = {},
 	state = State.TRACKER,
+	PCHealTrackingButtonState = 0,
```
Marked as resolved because this was just a comment rather than an action to resolve.

bdjeffyp commented:
Thanks, Fell! I appreciate it!

Fellshadow commented:
Update:

- Improved the auto-tracking toggle button to be similar to the settings toggle for better user clarity, I suppose the fact that this is for auto-tracking isn't clear on the tracker itself but there's only so much space to work with.
- Converted to a boolean instead of integer values for the state (which worked fine but I agree that a boolean makes more sense).
- Implemented manual tracking buttons that a user can use to manually add/remove PC heal counts if desired (to revert any accidental counts, for example).


![Screenshot_3](https://user-images.githubusercontent.com/106463662/173824489-5c98e2fb-1b0b-4c28-ac57-434e876892db.png)

Fellshadow commented:
additional update: replaced the chevrons with + and - signs for more clarity, though annoyingly the minus sign is off by half a pixel lol 
![image](https://user-images.githubusercontent.com/106463662/173830091-143dccbe-3da0-468b-91fe-a3329ed6363b.png)

Fellshadow commented on ironmon_tracker/Buttons.lua:
```diff
@@ -23,6 +23,16 @@ StatButtonColors = {
 	GraphicConstants.LAYOUTCOLORS.INCREASE
 }
 
+PCHealTrackingButtonStates = {
```
This has been implemented now 👍

Fellshadow commented on ironmon_tracker/Buttons.lua:
```diff
@@ -23,6 +23,16 @@ StatButtonColors = {
+	"+"
+}
+
+PCHealTrackingButtonColors = {
```
Table has now been removed due to rewriting of implementation
